lib/Doctrine/DBAL/ParameterType.php

```diff
@@ -49,6 +49,11 @@ final class ParameterType
      */
     public const BINARY = 16;
 
+    /**
+     * Represents a double data type.
+     */
+    public const DOUBLE = 32;
```

Comment:
It doesn't have to be a power of two since parameter types do aren't used as bits in a bitmask. Please use `17`.